apps/explorer/src/components/transactions/TransactionsForAddress.tsx

```diff
@@ -15,6 +15,49 @@ interface Props {
 	type: 'object' | 'address';
 }
 
+export function TransactionsForAddressTable({
```

Comment:
It's really not clear to me why this was split into its own component?

Reply:
This is used in the current implementation also. Unless we are okay to merge this PR and change the experimental route immediately to prod, the production code wont work after this is merged in